setup.py

```diff
@@ -25,24 +25,24 @@ def find_version(*paths):
 
 
 setuptools.setup(
-    name="deltacat",
+    name="deltacat-fork",
```

Comment:
Why renaming to deltacat-fork?

Reply:
Sorry this carried over from my fork version testing - I'll change it

Reply:
Good catch